examples/js/loaders/GLTFLoader.js

```diff
@@ -299,7 +299,8 @@ THREE.GLTFLoader = ( function () {
 
 		this.lights = {};
 
-		var lights = json.extensions && json.extensions[ EXTENSIONS.KHR_MATERIALS_COMMON ].lights;
+		var exts = json.extensions && json.extensions[ EXTENSIONS.KHR_MATERIALS_COMMON ];
+		var lights = ( exts && exts.lights ) || {};
```

Comment:
As long as we're here, I don't think that `json.extensions[ EXTENSIONS.KHR_MATERIALS_COMMON ]` is required either. `json.extensionsUsed` must contain the string `"KHR_materials_common"`, but AFAIK that's the only requirement.

So this might be best:

```
var extension = json.extensions && json.extensions[ EXTENSIONS.KHR_MATERIALS_COMMON ] || {};
var lights = extension.lights || {};
```